core/container.go

```diff
@@ -311,7 +325,7 @@ func (container *Container) From(ctx context.Context, gw bkgw.Client, addr strin
 	fsSt := llb.Image(
 		digested.String(),
 		llb.WithCustomNamef("pull %s", ref),
-		p.LLBOpt(),
+		p.LLBOpt(rec),
```

Comment:
Note to reviewer: these `LLBOpt`/`ProgressGroup` changes are a no-op now that I've made the PR backwards-compatible (by moving the `pipeline.Path` -> Progrock conversion to the Buildkit translation layer), but I left them in assuming we'd just be making the same changes in the future.